metadata-ingestion/tests/integration/lookml/test_lookml.py

```diff
@@ -141,6 +143,36 @@ def test_lookml_refinement_include_order(pytestconfig, tmp_path, mock_time):
     )
 
 
+@freeze_time(FROZEN_TIME)
+def test_lookml_explore_refinement(pytestconfig, tmp_path, mock_time):
+    looker_model = LookerModel(
+        explores=[
+            {
+                "name": "book",
+            },
+            {"name": "+book", "extends__all": [["order"]]},
+        ],
+        connection=str(),
+        resolved_includes=[],
+        includes=[],
+    )
+
+    refinement_resolver = LookerRefinementResolver(
+        looker_model=looker_model,
+        looker_viewfile_loader=None,  # type: ignore
+        reporter=None,  # type: ignore
+        source_config=SimpleNamespace(process_refinements=True),  # type: ignore
```

Comment:
what is this SimpleNamespace thing?

Reply:
To independently test this logic, I only need `process_refinements` config key, so instead of creating instance of `LookMLSourceConfig`, I passed this dynamic instance, which can be used in place of `LookMLSourceConfig`. 

To avoid confusion I updated this test-case and created the instance of `LookMLSourceConfig` by passing fake configuration

Reply:
thanks